components/Header/Header.tsx

```diff
@@ -29,7 +33,11 @@ const Header: React.FC = () => {
   };
   return (
     <>
-      <nav className="fixed inset-0 z-50 flex items-center justify-between px-10 bg-lightBg dark:bg-darkBg max-h-24">
+      <nav
+        className={`fixed inset-0 z-50 flex items-center justify-between px-10 bg-lightBg dark:bg-darkBg max-h-24 ${
+          locale === 'he' && 'flex-row-reverse'
+        }`}
+      >
```

Comment:
We have a  `useLocalStorage.ts` for that please use it.

Reply:
can u help me understand 2 things 
1. I can't use useLocalStorage in  server component Heders and layout  
2. what is the point of using local storage in the first place because `locale` is updated fine at my tests 
 can u enlight me what I am missing here

Reply:
Sorry my mistake here, i ment to say that we have a hook for language ditection (no idea why I was talking about localstorage)